src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AttendancesController.kt

```diff
@@ -88,6 +88,19 @@ class AttendancesController(private val attendanceService: AttendanceService) {
     )
   }
 
+  @PostMapping("/{prison}/attendance-over-date-range")
+  @ApiOperation(value = "Returns set of attendance details for set of booking ids", response = AttendancesResponse::class, notes = "Request attendance details")
+  fun getAttendanceForBookingsOverDateRangeByPost(@ApiParam(value = "Prison id (LEI)") @PathVariable(name = "prison") prisonId: String,
+                                                  @ApiParam(value = "Date of event in format YYYY-MM-DD", required = true) @RequestParam(name = "fromDate") @DateTimeFormat(iso = DATE) fromDate: LocalDate,
+                                                  @ApiParam(value = "Date of event in format YYYY-MM-DD defaults to fromDate") @RequestParam(name = "toDate") @DateTimeFormat(iso = DATE) toDate: LocalDate?,
+                                                  @ApiParam(value = "Time period. Leave blank for AM + PM") @RequestParam(name = "period") period: TimePeriod?,
+                                                  @ApiParam(value = "Booking ids (bookings=1&bookings=2)", required = true) @RequestBody bookings: Set<Long>): AttendancesResponse {
+
```

Comment:
Meh, example might be a bit misleading.

Reply:
Fixed, and updated on the other post route to match.